internal/featuresupport/feature_support_test.go

```diff
@@ -492,36 +505,39 @@ var _ = Describe("V2ListFeatureSupportLevels API", func() {
 				filters := SupportLevelFilters{OpenshiftVersion: "4.12", CPUArchitecture: &cpuArchitecture}
 				Expect((&MinimalIso{}).getSupportLevel(filters)).To(Equal(models.SupportLevelSupported))
 			})
-			It("Empty support level - platforms", func() {
-				for _, platform := range []models.PlatformType{models.PlatformTypeOci, models.PlatformTypeVsphere, models.PlatformTypeNutanix, models.PlatformTypeBaremetal, models.PlatformTypeNone, models.PlatformTypeExternal} {
-					p := platform
-
-					for _, feature := range []SupportLevelFeature{&VsphereIntegrationFeature{}, &NutanixIntegrationFeature{},
-						&BaremetalPlatformFeature{}, &NonePlatformFeature{}, &OciIntegrationFeature{}, &ExternalPlatformFeature{}} {
-						f := feature
 
-						By(fmt.Sprintf("Feature %s", f.GetName()), func() {
-							filters := SupportLevelFilters{OpenshiftVersion: "", CPUArchitecture: nil, PlatformType: nil}
-							Expect(string(f.getSupportLevel(filters))).To(Not(Equal("")))
-
-							filters = SupportLevelFilters{OpenshiftVersion: "", CPUArchitecture: nil, PlatformType: &p}
-							Expect(string(f.getSupportLevel(filters))).To(Equal(""))
+			for _, filters := range getPlatformFilters() {
```

Comment:
Not directly related to this PR, but wouldn't it be worth switching to table ginkgo testing?

Reply:
Thanks, I'll leave it like that for the moment, at least the test does test something now. But I agree that we need to use tables.